cloudfoundry-operations/src/main/java/org/cloudfoundry/operations/applications/DefaultApplications.java

```diff
@@ -566,29 +564,28 @@ public Mono<Boolean> sshEnabled(ApplicationSshEnabledRequest request) {
     @Override
     public Mono<Void> start(StartApplicationRequest request) {
         return Mono
-            .zip(this.cloudFoundryClient, this.spaceId)
-            .flatMap(function((cloudFoundryClient, spaceId) -> Mono.zip(
-                Mono.just(cloudFoundryClient),
-                getApplicationIdWhere(cloudFoundryClient, request.getName(), spaceId, isNotIn(STARTED_STATE))
-            )))
-            .flatMap(function((cloudFoundryClient, applicationId) -> startApplicationAndWait(cloudFoundryClient, request.getName(), applicationId, request.getStagingTimeout(),
-                request.getStartupTimeout())))
-            .transform(OperationsLogging.log("Start Application"))
-            .checkpoint();
+                .zip(this.cloudFoundryClient, this.spaceId)
```

Comment:
Use this link http://v3-apidocs.cloudfoundry.org/version/3.130.0/index.html#starting-apps for start() implementation